src/audio/module_adapter/module_adapter.c

```diff
@@ -230,7 +230,7 @@ int module_adapter_prepare(struct comp_dev *dev)
 	list_for_item(blist, &dev->bsink_list)
 		mod->num_output_buffers++;
 
-	if (!mod->num_input_buffers || !mod->num_output_buffers) {
+	if (!mod->num_input_buffers && !mod->num_output_buffers) {
```

Comment:
can we change the err message below to say that both input and output buffer counts are 0?

Reply:
Sure